include/tvm/tir/op.h

```diff
@@ -762,6 +778,15 @@ inline PrimExpr make_zero(DataType t) {
   }
   return make_const(t, 0);
 }
+
+template <typename FReduce>
+inline PrimExpr foldl(FReduce freduce, PrimExpr init_value, const Array<PrimExpr>& values) {
```

Comment:
@tqchen I don't have strong opinion, but is it slightly better to use `Foldl` instead to conform the naming convention?

Reply:
most of the functions in this file uses stl_case(as they are part of user facing function), we can discuss whether we want to force all function to conform to the CamelCase later.

Reply:
Okay, it makes sense :-)